src/main/java/com/connexta/poller/service/TransformPollingTask.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.poller.service;
+
+import com.connexta.poller.api.PollingTask;
+import com.dyngr.core.AttemptResult;
+import java.net.URI;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * This polling task implements the logic to contact the transform status endpoint and receive
+ * status information about a specific job. The polling task will complete when the transform job
+ * reaches the complete or failed status. The polling task will also end if the PollingService
+ * exceeds the services maximum number of number of retries or reaches other service-specific
+ * thresholds.
+ *
+ * @param <T>
+ */
+public class TransformPollingTask<T> implements PollingTask<T> {
```

Comment:
Same question about the API.

Reply:
https://github.com/connexta/ion-store/pull/223/files#r343311328